docs/security_self_assessment.md

```diff
@@ -0,0 +1,1588 @@
+
+docker inspect etcd | jq -r '.[].HostConfig.Binds[]' | grep "${test_dir}" | cut -d ":" -f 1 | xargs stat -c %a
+```
+
+Audit Execution:
+
+`./1.1.11.sh etcd`
+
+Expected result:
+
+    '700' is equal to '700'
+
+*1.1.12* Ensure that the etcd data directory ownership is set to `etcd:etcd` (Scored)
+
+Result: Not Applicable
+
+Remediation: On the etcd server node, get the etcd data directory, passed as an argument --data-dir, from the below command:
+
+`ps -ef | grep etcd`
+
+Run the below command (based on the etcd data directory found above). For example,
+
+`chown etcd:etcd /var/lib/etcd`
+
+Audit Script: `1.1.12.sh`
+
+```sh
+#!/bin/bash -e
+
+etcd_bin=${1}
+
+test_dir=$(ps -ef | grep ${etcd_bin} | grep -- --data-dir | sed 's%.*data-dir[= ]\([^ ]*\).*%\1%')
+
+docker inspect etcd | jq -r '.[].HostConfig.Binds[]' | grep "${test_dir}" | cut -d ":" -f 1 | xargs stat -c %U:%G
+```
+
+Audit Execution:
+
+`./1.1.12.sh etcd`
+
+Expected result:
+
+    'etcd:etcd' is present
+
+*1.1.13* Ensure that the admin.conf file permissions are set to 644 or more restrictive (Scored)
+
+Result: Not Applicable
+
+Remediation: RKE2 does not store the kubernetes default kubeconfig credentials file on the nodes. It’s presented to user where RKE2 is run. We recommend that this `kube_config_cluster.yml` file be kept in secure store.
+
+*1.1.14* Ensure that the admin.conf file ownership is set to `root:root` (Scored)
+
+Result: Not Applicable
+
+Remediation: RKE2 does not store the kubernetes default kubeconfig credentials file on the nodes. It’s presented to user where RKE2 is run. We recommend that this `kube_config_cluster.yml` file be kept in secure store.
+
+*1.1.15* Ensure that the scheduler.conf file permissions are set to `644` or more restrictive (Scored)
+
+Result: Not Applicable
+
+Remediation: RKE2 doesn’t require or maintain a configuration file for the scheduler. All configuration is passed in as arguments at container run time.
+
+*1.1.16* Ensure that the scheduler.conf file ownership is set to `root:root` (Scored)
+
+Result: Not Applicable
+
+Remediation: RKE2 doesn’t require or maintain a configuration file for the scheduler. All configuration is passed in as arguments at container run time.
+
+*1.1.17* Ensure that the controller-manager.conf file permissions are set to `644` or more restrictive (Scored)
+
+Result: Not Applicable
+
+Remediation: RKE2 doesn’t require or maintain a configuration file for the controller manager. All configuration is passed in as arguments at container run time.
+
+*1.1.18* Ensure that the controller-manager.conf file ownership is set to `root:root` (Scored)
+
+Result: Not Applicable
+
+Remediation: RKE2 doesn’t require or maintain a configuration file for the controller manager. All configuration is passed in as arguments at container run time.
+
+*1.1.19* Ensure that the Kubernetes PKI directory and file ownership is set to `root:root` (Scored)
+
+Result: Not Applicable
```

Comment:
It is applicable. See https://github.com/rancher/security-scan/pull/22/files#r493609155